README.md

```diff
@@ -20,6 +24,9 @@ substituters = http://nixcache.kadena.io https://pact.cachix.org https://nixcach
 trusted-public-keys = kadena-cache.local-1:8wj8JW8V9tmc5bgNNyPM18DYNA1ws3X/MChXh1AQy/Q= pact.cachix.org-1:cg1bsryGrHnQzqEp52NcHq4mBBL+R25XbR2Q/I/vQ8Y= ryantrinkle.com-1:JJiAKaRv9mWgpVAz8dwewnZe0AzzEAzPkagE9SP5NWI= cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
 ```
 
+This will allow you to connect to Kadena's Nix cache, and pull pre-built
```

Comment:
Do we really want to be on the hook for supporting the world's chainweb nix builds?

Reply:
You tell me :)

Reply:
Yes, I think this is perfectly reasonable.  Supplying a nix cache is basically equivalent to supplying binaries.

Reply:
And with the CDN plan, this should be even easier.